keyboards/handwired/oem_ansi_fullsize/config.h

```diff
@@ -0,0 +1,187 @@
+/*
+Copyright 2021 Cian Johnston
+
+This program is free software: you can redistribute it and/or modify
+it under the terms of the GNU General Public License as published by
+the Free Software Foundation, either version 2 of the License, or
+(at your option) any later version.
+
+This program is distributed in the hope that it will be useful,
+but WITHOUT ANY WARRANTY; without even the implied warranty of
+MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+GNU General Public License for more details.
+
+You should have received a copy of the GNU General Public License
+along with this program.  If not, see <http://www.gnu.org/licenses/>.
+*/
+
+#pragma once
+
+#include "config_common.h"
+
+/* USB Device descriptor parameter */
+#define VENDOR_ID 0xFEED
+#define PRODUCT_ID 0x0000
+#define DEVICE_VER 0x0001
+#define MANUFACTURER Cian Johnston
+#define PRODUCT oem_ansi_fullsize
+
+/* key matrix size */
+#define MATRIX_ROWS 6
+#define MATRIX_COLS 22
+
+/*
+ * Keyboard Matrix Assignments
+ *
+ * Change this to how you wired your keyboard
+ * COLS: AVR pins used for columns, left to right
+ * ROWS: AVR pins used for rows, top to bottom
+ * DIODE_DIRECTION: COL2ROW = COL = Anode (+), ROW = Cathode (-, marked on diode)
+ *                  ROW2COL = ROW = Anode (+), COL = Cathode (-, marked on diode)
+ *
+ *
+ *  Teensy++ 2.0 Pin Layout
+ *
+ *                   ┌──────┐
+ *              ┌────┴──────┴────┐
+ *          +5V │o              o│ GND
+ *    CL <== B6 │o              o│ B7 ==> CE
+ *    CK <== B5 │o              o│ D0 ==> CD
+ *    CJ <== B4 │o              o│ D1 ==> CC
+ *    CI <== B3 │o              o│ D2 ==> CB
+ *    CH <== B2 │o              o│ D3 ==> CA
+ *    CG <== B1 │o              o│ D4 ==> C9
+ *    CF <== B0 │o              o│ D5 ==> C8
+ *           E7 │o              o│ D6 ==> NOPE
+ *    C7 <== E6 │o              o│ D7 ==> C6
+ *          GND │o              o│ E0 ==> C5
+ *          RST │o              o│ E1 ==> C4
+ *    R5 <== F0 │o              o│ C0 ==> C3
+ *    R4 <== F1 │o  A0 o  o A4  o│ C1 ==> C2
+ *    R3 <== F2 │o  A1 o  o A5  o│ C2 ==> C1
+ *    R2 <== F3 │o  A2 o  o A6  o│ C3 ==> C0
+ *    R1 <== F4 │o  A3 o  o A7  o│ C4
+ *    R0 <== F5 │o              o│ C5
+ *           F6 │o              o│ C6
+ *           F7 │o  o   o   o   o│ C7
+ *              └────────────────┘
+ *                  R   G   +
+ *                  S   N   5
+ *                  T   D   V
+ */
```

Comment:
This probably belongs more in the readme, tbh.

Reply:
Done